apps/src/p5lab/P5LabVisualizationHeader.jsx

```diff
@@ -47,6 +49,7 @@ class P5LabVisualizationHeader extends React.Component {
       mode === P5LabInterfaceMode.ANIMATION ||
       mode === P5LabInterfaceMode.BACKGROUND
     ) {
+      this.props.setAnimationType(mode);
```

Comment:
it looks like we set the interface mode at the same time as the animation type. Is there a benefit to having both? I see animation type only includes `ANIMATION` or `BACKGROUND`, when we go back to code do we need to know the animation type still?

Reply:
We don’t need to know the animation type in Code mode because it isn’t used. I initialized it to “default”, but I could just do an empty string and reset to that when switching to Code?